Source/Core/Core/IOS/Network/IP/Top.cpp

```diff
@@ -258,8 +254,8 @@ static std::optional<DefaultInterface> GetSystemDefaultInterface()
 
 static DefaultInterface GetSystemDefaultInterfaceOrFallback()
 {
-  static constexpr DefaultInterface FALLBACK_VALUES{
-      inet_addr(10, 0, 1, 30), inet_addr(255, 255, 255, 0), inet_addr(10, 0, 255, 255)};
+  // Fallback values in little endian: {10.0.1.30, 255.255.255.0, 10.0.255.255}
+  static constexpr DefaultInterface FALLBACK_VALUES{0x0e01000a, 0x00ffffff, 0xffff000a};
```

Comment:
Couldn't you just change our `inet_addr` to use a different endianness (or just add a call to Common::Swap32)? This is less readable.

Reply:
I took a similar "swap" approach using the native `inet_addr` as it returns the value as expected by the DefaultInterface structure.